utilities/project_planning/calculate_available_weeks.py

```diff
@@ -0,0 +1,67 @@
+"""
+Calculate max weeks of programmer availability for yearly planning.
+
+Assumes the following:
+
+1. The number of maintainers stays the same
+2. We cannot anticipate availability changes like parental leave, part-time moves, etc.
+3. 5 weeks of holiday + sick leave with stipulation that this is just an estimate!!!!!
+4. Not all conferences are accounted for
+"""
+
+programmers_with_reduced_availability = {
+    # _ is part-time (safer to assume that isn't going to change)
+    "_": (52 / 2) * 0.6,
+    # __ is on parental leave for the first quarter of the year
+    "__": 52 - (52 / 4),
+    # assume whoever is lead at any point probably has 1/3rd availability for dev work
+    "lead": 52 / 3,
+}
+
+total_openverse_maintainers = 8
+
+programmers_with_regular_availability = {
+    "-" * (i + 1): 52
+    for i in range(
+        total_openverse_maintainers - len(programmers_with_reduced_availability)
+    )
+}
+
+programmer_availability_ignoring_regular_leave = (
+    programmers_with_reduced_availability | programmers_with_regular_availability
+)
+
+# These are just vague estimates, obviously some folks take more or less, etc.
+holiday_weeks = 4
```

Comment:
@AetherUnbound are you able to see the sponsored contributors average AFK for this year and calculate based on that? I'm worried 4 might be too low :thinking: If the average from this year is lower though then let's keep at 4 to assume the sponsored contributors will (correctly :slightly_smiling_face:) take the leave they're expected to.

Reply:
I think I can, I'll take a look and adjust if it seems inaccurate!

Reply:
Update: this number is accurate.